db/migrate/20150410161434_create_questions.rb

```diff
@@ -1,6 +1,10 @@
 class CreateQuestions < ActiveRecord::Migration
   def change
     create_table :questions do |t|
+      t.references :creator, class_name: 'User'
+      t.string :title
+      t.string :content
+      t.references :best_answer, class_name: 'Answer'
 
```

Comment:
The class_name is needed on the model but not on the migration - the database doesn't know anything about your classes.